src/Symfony/Component/Mailer/Bridge/Mailjet/Transport/MailjetApiTransport.php

```diff
@@ -177,4 +194,19 @@ private function getEndpoint(): ?string
     {
         return ($this->host ?: self::HOST).($this->port ? ':'.$this->port : '');
     }
+
+    private function castCustomHeader(string $value, string $type)
+    {
+        switch ($type) {
+            case 'bool':
+                return (bool) $value;
```

Comment:
could it make sense to use `filter_var($value, \FILTER_VALIDATE_BOOLEAN)`?

Reply:
I think it makes sense since user can set bool headers as
```
$headers->addTextHeader('X-MJ-TemplateLanguage', 'on');
$headers->addTextHeader('X-MJ-TemplateLanguage', 'true');
$headers->addTextHeader('X-MJ-TemplateLanguage', 'yes');
```